app/assets/stylesheets/bootstrap-overrides/bootstrap-overrides.scss

```diff
@@ -3,3 +3,7 @@
  * Variables that can be overridden can be found at:
  * https://github.com/twbs/bootstrap-sass/blob/v3.3.6/assets/stylesheets/bootstrap/_variables.scss
  */
+
```

Comment:
The way `bootstrap-overrides.scss` is being used right now is not how I intended it when I created it (https://github.com/on-site/grantzilla/commit/7dcaf9a4e48dbc52a2e092794c68e1bddc1502ce).  It really makes sense to create a new scss file for the content below, with some more appropriate name, because this file needs to be here specifically for the case where we want to override any variables defined by in the bootstrap library.